### PR TITLE
Update to the latest elastic-package to pick up the  package spec with optional datastream

### PR DESCRIPTION
**Pull request by aleksmaus**
Picks up the https://github.com/elastic/elastic-package/pull/232

### CODE REVIEW

File: go.sum [hunk 1]
```diff
@@ -84,23 +103,27 @@ github.com/davecgh/go-spew v1.1.1 h1:vj9j/u1bqnvCEfJOwUhtlOARqs3+rkHYY13jYWTU97c
 github.com/davecgh/go-spew v1.1.1/go.mod h1:J7Y8YcW2NihsgmVo/mv3lAwl/skON4iLHjSsI+c5H38=
 github.com/dgrijalva/jwt-go v3.2.0+incompatible/go.mod h1:E3ru+11k8xSBh+hMPgOLZmtrrCbhqsmaPHjLKYnJCaQ=
 github.com/dgryski/go-sip13 v0.0.0-20181026042036-e10d5fee7954/go.mod h1:vAd38F8PWV+bWy6jNmig1y/TA+kYO4g3RSRF0IAv0no=
-github.com/elastic/elastic-package v0.0.0-20210118134226-ec5c00a8ad4d h1:KYFFjqV2c2Yz1XL1815XyB4omlDd2597ua/usgqCVcI=
-github.com/elastic/elastic-package v0.0.0-20210118134226-ec5c00a8ad4d/go.mod h1:lphRg8ZgCi63Nd3vpKy29oqx2N/mme2VdTSHpN1ZNnk=
+github.com/elastic/elastic-package v0.0.0-20210125150915-a522f84a2a2a h1:QHsWIAAZV1JOgk/sLChSWc2ywRcIPRkD+5EnOiQqQxA=
+github.com/elastic/elastic-package v0.0.0-20210125150915-a522f84a2a2a/go.mod h1:+3qq7Ovx+4JJ6JoRnQThRs51UiiY3m+QMfoDDjo1BHI=
 github.com/elastic/go-elasticsearch/v7 v7.9.0 h1:UEau+a1MiiE/F+UrDj60kqIHFWdzU1M2y/YtBU2NC2M=
 github.com/elastic/go-elasticsearch/v7 v7.9.0/go.mod h1:OJ4wdbtDNk5g503kvlHLyErCgQwwzmDtaFC4XyOxXA4=
+github.com/elastic/go-elasticsearch/v7 v7.10.0 h1:vYRwqgFM46ZUHFMRdvKr+y1WA4ehJO6WqAGV9Btbl2o=
```
Comment (ruflin) [inline go.sum:110]: something is off here? `tidy` needed?

Comment (aleksmaus) [reply go.sum:110]: `go mod tidy` doesn't remove this line

Comment (aleksmaus) [reply go.sum:110]: did run 
```go get -u github.com/elastic/elastic-package```
and 
```mage check```

Comment (ruflin) [reply go.sum:110]: Did not check it out locally but I guess it is related to all the changes in the go.mod file. @mtojek might know more.

**Comment by ycombinator** [comment]
@aleksmaus Comparing with https://github.com/elastic/integrations/blob/master/go.mod, it looks like most of the updates in the `go.mod` in this PR are for indirect dependencies (marked by the `// indirect` comment at the end). Would you mind updating this PR to remove those indirect dependencies from `go.mod` and then re-running `go mod tidy`? Let's see if that reduces the "noise" in this PR. Thanks!

**Comment by aleksmaus** [comment]
@ycombinator done

**Review by ycombinator — APPROVED** [review]
LGTM. WFG.

**Pull request merged**